### DISCUSSION TITLE
ScriptReference/PlayerSettings.GetApiCompatibilityLevel

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#15] [Notes]
# ScriptReference/PlayerSettings.GetApiCompatibilityLevel

https://docs.unity3d.com/ScriptReference/PlayerSettings.GetApiCompatibilityLevel.html

**Comment by vertxxyz** [comment]
You can use [`EditorUserBuildSettings.selectedBuildTargetGroup`](https://docs.unity3d.com/ScriptReference/EditorUserBuildSettings-selectedBuildTargetGroup.html) to get the current selected build target.
